packages/client/src/src-sw.js

```diff
@@ -71,7 +71,7 @@ workbox.precaching.precacheAndRoute([])
  */
 
 workbox.routing.registerRoute(
-  '/config.js',
+  /http(.+)config\.js$/,
```

Comment:
Not the best regexp, I know. There was a tiny undocumented constraint with the `registerRoute` – it only lets through cross-domain URLs if the regexp given matches the url from the beginning, so from position 0.

https://github.com/GoogleChrome/workbox/blob/0cc6975f17b60d67a71d8b717e73ef7ddb79a891/packages/workbox-routing/src/RegExpRoute.ts#L69

Reply:
I think this is something that prevented the app from working in offline completely as it is now

Reply:
Amazing!